### DISCUSSION TITLE
Welcome to XRechnung-for-Delphi Discussions!

### DISCUSSION THREAD

**Discussion by landrix** [LandrixSoftware/XRechnung-for-Delphi#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PascalRiekenberg** [comment]
Hello, i am wondering if this works with FPC/Lazarus.

**Reply by landrix** [reply]
Any platform? or Windows only? i can take a look on this.

**Reply by PascalRiekenberg** [reply]
Would be great to have it platform independant but Windows only would be a start. Thanks.

**Reply by landrix** [reply]
i've greated an issue fot that
